shopfloor_checkout_package_measurement/services/checkout.py

```diff
@@ -61,7 +63,11 @@ def set_package_measurement(
 
     def _put_lines_in_allowed_package(self, picking, selected_lines, package):
         res = super()._put_lines_in_allowed_package(picking, selected_lines, package)
-        if not picking._check_required_package_measurement(package=package):
+        try:
+            package.with_context(
```

Comment:
See my comment on the other PR.
Also, never override the whole ctx unless you really want to. Use kw arg instead.

Reply:
fixed